CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ### Added
 - Support for Golang fileops
+- Intellij Extension for mirrord
```

Comment:
Note that it's styled IntelliJ (not too important here, but for future user-facing PRs)